.infrastructure/pv.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: todoapp-pv
+  namespace: todoapp
+spec:
+  storageClassName: standard
+  persistentVolumeReclaimPolicy: Retain
+  accessModes: ["ReadWriteMany"]
+  capacity: 
+    storage: 1Gi
+  hostPath: 
+    path: /data/
```

Comment:
Using 'hostPath' for PersistentVolume is generally not recommended for production environments as it ties the volume to a specific node. Consider using a different volume type, such as NFS or a cloud provider's volume, if this is intended for production.